examples/p5js/Handpose/Handpose_Part_Selection/index.html

```diff
@@ -0,0 +1,13 @@
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Handpose with Webcam</title>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/p5.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/p5.js/0.9.0/addons/p5.dom.min.js"></script>
+    <script src="http://localhost:8080/ml5.js" type="text/javascript"></script>
+  </head>
+  <body>
+    <h1>Handpose with Webcam</h1>
```

Comment:
How about we update this title to something with 'Handpose with Webcam (Part Selection)'? Might make this easier to differentiate from the other `Handpose_Webcam` example.